modules/diskq/qdisk.c

```diff
@@ -497,20 +497,21 @@ _try_to_load_queue(QDisk *self, GQueue *queue, LoadParameters *params)
    .type = #_type \
 }
 
+#define try_load_queue(queue) \
+( { \
+  gboolean retval = TRUE; \
+  LoadParameters queue ## _parameters = create_load_parameter_for(queue); \
+  if (!_try_to_load_queue(self, queue, &queue ## _parameters)) \
+    retval = FALSE; \
+  retval; \
+} )
+
```

Comment:
hmmmm....  I'd suggest to follow a different approach here.
What if we would have three different functions for loading `qout`, `qoverflow` and `qbacklog`?
Or, we could group the variables per queue into a type... and the loader could receive pointer to an instance with that type?

Reply:
Fixed.